### DISCUSSION TITLE
Ordering List Problem

### DISCUSSION THREAD

**Discussion by xcjwq** [sharadcodes/jekyll-theme-serial-programmer#12] [Q&A / Get Help]
Hey um, I need help, I'm not experienced in web development.

Ordered list went error, can't display correctly beginning from number 10  
![image](https://user-images.githubusercontent.com/19588033/189567518-49e2621b-20a0-4b45-8fd4-9048d9ccc4fd.png)

And also the emoji is dislocated  
![image](https://user-images.githubusercontent.com/19588033/189567762-b46a9435-0fb1-4fe1-9e59-11c95bc6dfeb.png)

Thanks!

**Comment by sharadcodes** [comment]
Hi @wyebit 
Can you share the markdown code ?
It's probably because of the spacing issue, but I'll have to see it for myself.

**Reply by xcjwq** [reply]
https://github.com/wyebit/wyebit.github.io/blob/main/all_collections/_posts/2022-09-11-create-wordpress-based-website-with-mysql-database.md

Here's my markdown code

**Reply by sharadcodes** [reply]
I've fixed css for emoji, I'll see what's wrong with the list, it's a Jekyll problem.

**Reply by xcjwq** [reply]
> I've fixed css for emoji, I'll see what's wrong with the list, it's a Jekyll problem.

Is there something that I can do for the list? Temporary fix maybe?

By the way, thanks for fixing the emoji problem.

**Reply by sharadcodes** [reply]
I'll tell you if I come across something

**Reply by xcjwq** [reply]
Okay, looking forward to it
